keyboards/anavi/macropad8/macropad8.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_ortho_2x4( \
+    k01, k02, k03, k04, \
+    k05, k06, k07, k08  \
+  ) { \
+    { k01, k02, k03, k04 }, \
+    { k05, k06, k07, k08 } \
+}
+
+#define LAYOUT LAYOUT_ortho_2x4 
```

Comment:
I would very much prefer to not have the layout macro alias here, and having the codebase reference `LAYOUT_ortho_2x4` directly.

```suggestion
```